lib/property_sets/active_record_extension.rb

```diff
@@ -184,6 +184,38 @@ def update_property_set_attributes(attributes)
           end
         end
       end
+
+      def update_columns(attributes)
+        attrs = if delegated_property_sets?
+          attributes.reject{|k,_| self.class.delegated_property_set_attributes.include?(k.to_s) }
+        else
+          attributes
+        end
+
+        raise ArgumentError, "Empty list of attributes to change" if attrs.empty?
+        super attrs
+      end
+
+      private
+
+      def delegated_property_sets?
+        self.class.respond_to?(:delegated_property_set_attributes)
+      end
+
+      def attributes_for_create(attribute_names)
+        super filter_delegated_property_set_attributes(attribute_names)
+      end
+
+      def attributes_for_update(attribute_names)
+        super filter_delegated_property_set_attributes(attribute_names)
+      end
+
+      def filter_delegated_property_set_attributes(attribute_names)
+        if delegated_property_sets?
+          return attribute_names.map(&:to_s) - self.class.delegated_property_set_attributes.to_a
```

Comment:
this is changing the input from symbols to strings, the update_columns was better in this regard since it only rejected

Reply:
The `.map(&:to_s)` was only a defensive coercion in case future rails changed behavior. But I can pull that out.

Reply:
yeah either way is meh ... it's slow (using reject) or converts input (using to_s) or might fail later when not converting ... so 🤷 pick whatever you like :D